nipype/workflows/smri/freesurfer/utils.py

```diff
@@ -371,3 +374,489 @@ def create_tessellation_flow(name='tessellate', out_format='stl'):
         ])
     return tessflow
+
+def copy_files(in_files, out_files):
+    """
+    Create a function to copy a file that can be modified by a following node 
+    without changing the original file
+    """
+    import shutil
+    import sys
+    if len(in_files) != len(out_files):
+        print "ERROR: Length of input files must be identical to the length of \
+        outrput files to be copied"
+        sys.exit(-1)
+    for i, in_file in enumerate(in_files):
+        out_file = out_files[i]
+        print "copying %s to %s" % (in_file, out_file)
+        shutil.copy(in_file, out_file)
+    return out_files
+
+def copy_file(in_file, out_file=None):
+    """
+    Create a function to copy a file that can be modified by a following node 
+    without changing the original file.
+    """
+    import os
+    import shutil
+    if out_file == None:
+        out_file = os.path.join(os.getcwd(), os.path.basename(in_file))
+    if type(in_file) is list and len(in_file) == 1:
+        in_file = in_file[0]
+    out_file = os.path.abspath(out_file)
+    in_file = os.path.abspath(in_file)
+    print "copying %s to %s" % (in_file, out_file)
+    shutil.copy(in_file, out_file)
+    return out_file
+
+def mkdir_p(path):
+    import errno
+    import os
+    try:
+        os.makedirs(path)
+    except OSError as exc:  # Python >2.5
+        if exc.errno == errno.EEXIST and os.path.isdir(path):
+            pass
+        else:
+            raise
+        
+def getdefaultconfig():
+    config = { 'custom_atlas' : None,
+               'cw256' : False,
+               'field_strength' : '1.5T',
+               'fs_home' : checkenv(),
+               'longitudinal' : False,
+               'long_base' : None,
+               'openmp' : None,
+               'plugin_args' : None,
+               'qcache' : False,
+               'queue' : None,
+               'recoding_file' : None,
+               'src_subject_id' : 'fsaverage'}
+
+    config['src_subject_dir'] = os.path.join(config['fs_home'], 'subjects',
+                                             config['src_subject_id'])
+    config['awk_file'] = os.path.join(config['fs_home'], 'bin',
+                                      'extract_talairach_avi_QA.awk')
+    config['registration_template'] = os.path.join(config['fs_home'], 'average',
+                                                   'RB_all_2014-08-21.gca')
+    config['registration_template_withskull'] = os.path.join(config['fs_home'], 'average',
+                                                             'RB_all_withskull_2014-08-21.gca')
+    for hemi in ('lh', 'rh'):
+        config['{0}_atlas'.format(hemi)] = os.path.join(
+            config['fs_home'], 'average',
+            '{0}.average.curvature.filled.buckner40.tif'.format(hemi))
+        config['{0}_classifier'.format(hemi)] = os.path.join(
+            config['fs_home'], 'average',
+            'rh.curvature.buckner40.filled.desikan_killiany.2010-03-25.gcs'.format(hemi))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Too many arguments for format string](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/2302210766/issues/source?bid=2767207&fileBranchId=3084409#l451)